src/main/java/atas/model/ModelManager.java

```diff
@@ -345,7 +345,12 @@ public String getSessionDetails() {
             requireNonNull(currentEnteredSession);
             String sessionName = currentEnteredSession.getSessionName().toString();
             String sessionDate = currentEnteredSession.getSessionDate().toString();
-            return String.format("Current Session: %s    Date: %s", sessionName, sessionDate);
+            String presenceStats = currentEnteredSession.getSessionStats()
+                .getPresenceStatistics().getDataAsPercentage();
+            String participationStats = currentEnteredSession.getSessionStats()
+                .getParticipationStatistics().getDataAsPercentage();
+            return String.format("Current Session: %s   Date: %s        %s    %s",
+                sessionName, sessionDate, presenceStats, participationStats);
```

Comment:
Should presenceStats and participationStats align to the right of the footer?

Reply:
Noted with thanks!